extras/config.ini

```diff
@@ -7,6 +7,7 @@ socket_buffer_size = 16384
 ;tls_cert = /etc/avalanchemq/cert.pem
 ;tls_key = /etc/avalanchemq/key.pem
 ;tls_compability_mode = false
+allow_guest_user_remotely = false
```

Comment:
what about? 

```suggestion
guest_only_loopback = true
```

Reply:
Yes probably better to reflect what the check actually does.